vsintegration/src/FSharp.LanguageService/ProjectSitesAndFiles.fs

```diff
@@ -140,36 +245,44 @@ type internal ProjectSitesAndFiles() =
             | _ -> Seq.empty
         | Some _ -> Seq.empty
 
-    static let rec referencedProvideProjectSites (projectSite:IProjectSite, serviceProvider:System.IServiceProvider) =
-        seq { 
+    static let rec referencedProvideProjectSites(projectSite:IProjectSite, serviceProvider:System.IServiceProvider, extraProjectInfo:obj option, projectOptions) =
+        seq {
             let solutionService = try Some (serviceProvider.GetService(typeof<SVsSolution>) :?> IVsSolution) with _ -> None
             match solutionService with
             | Some solutionService ->
                 for p in referencedProjects projectSite do
                     match solutionService.GetProjectOfUniqueName(p.UniqueName) with
                     | VSConstants.S_OK, (:? IProvideProjectSite as ps) ->
-                        yield (p, ps)
+                        yield (p.FileName,  (fullOutputAssemblyPath p) |> Option.defaultValue "", ps)
                     | _ -> ()
-            | None -> ()
+            | None ->
```

Comment:
This is dubious - your new code is only being hit when ``solutionService `` is ``None``, which should never happen?